### ISSUE TITLE
RetrieveModelDetailsAsync has an overload that takes an auth parameter it ignores

### ISSUE THREAD

**Issue by KeithHenry** [OkGoDoIt/OpenAI-API-dotnet#42]
Here:
https://github.com/OkGoDoIt/OpenAI-API-dotnet/blob/97fb71b7690c2e3b763ae45b1cc02b2133bb3e55/OpenAI_API/Model/ModelsEndpoint.cs#L48

**Comment by OkGoDoIt** [comment]
Good catch.  This is leftover from a previous implementation before refactoring.  I have implemented your suggested fix in https://github.com/OkGoDoIt/OpenAI-API-dotnet/commit/7755f4ceb08255ff5084b804184a476dd0dac050
